app_dart/lib/src/service/datastore.dart

```diff
@@ -142,14 +139,11 @@ class DatastoreService {
   Stream<FullTask> queryRecentTasks({
     String? taskName,
     int commitLimit = 20,
-    int taskLimit = 20,
     String? branch,
     required RepositorySlug slug,
   }) async* {
     await for (Commit commit in queryRecentCommits(limit: commitLimit, branch: branch, slug: slug)) {
-      final Query<Task> query = db.query<Task>(ancestorKey: commit.key)
-        ..limit(taskLimit)
```

Comment:
TIL we've had this for 3 years, and I don't think anything used it

Reply:
`refresh_chromebot_status` is using it as well.